src/wizards/tokenSwapDealWizard/components/stageButtons/stageButtons.html

```diff
@@ -9,8 +9,10 @@
     <pbutton
       if.bind="!showSubmit"
       type="primary"
-      click.delegate="wizardService.proceed(wizardManager)"
-      data-test="wizard-proceed-button">
+      click.delegate="proceed()"
+      data-test="wizard-proceed-button"
+      is-loading.bind="validating"
```

Comment:
why call this "is-loading" if is related to validating?  Why do we need this?

Reply:
I added this just to let the user know the app is doing something(validation in this case) before going to the next stage. If we don't have this loading icons, the user won't know why the "Proceed" button takes 2-3 seconds to change to the next stage. I am open to other suggestions.

Reply:
I get it, ok